### DISCUSSION TITLE
How to execute tctl command from a leaf cluster to the root cluster

### DISCUSSION THREAD

**Discussion by mukitaro** [gravitational/teleport#23429] [Q&A] [answered]
Hi,
I would like to execute tctl command to the root cluster from a node registered in the leaf cluster.
Teleport is using 12.0.4.

I have performed a login on the leaf cluster with the following command
```
$ tsh login --proxy=root_cluster:443 --auth=local --user=admin
```

If I try to view the list of users in the above state, for example, I will get an error.
```
$ tctl users ls
ERROR: tctl must be either used on the auth server or provided with the identity file via --identity flag
```
However, when I run the same command as above from my local PC, the list of users is displayed without any error.

Why is the tctl command not available from nodes registered in the leaf cluster?

**Comment by webvictim** [comment, marked as answer]
That looks to me like `tctl` is picking up the local `/etc/teleport.yaml` file rather than using your logged in `tsh` identity.

Try this: `TELEPORT_CONFIG_FILE="" tctl users ls`

**Reply by mukitaro** [reply]
Thank you.
I solved the problem by the method you taught me.
